packages/aragon-cli/src/helpers/ensureDevchain.js

```diff
@@ -1,21 +1,21 @@
 import path from 'path'
```

Comment:
Move to `aragon-cli` package using `aragon devchain` instead of `@aragon/aragen`.